orchestrator/crates/orchestrator/src/tests/jobs/state_update_job/mod.rs

```diff
@@ -327,9 +391,30 @@ async fn process_job_invalid_input_gap_panics() {
         .build()
         .await;
 
-    let mut metadata: HashMap<String, String> = HashMap::new();
-    metadata.insert(String::from(JOB_METADATA_STATE_UPDATE_BLOCKS_TO_SETTLE_KEY), String::from("6, 7, 8"));
-    metadata.insert(String::from(JOB_PROCESS_ATTEMPT_METADATA_KEY), String::from("0"));
+    // Create proper metadata structure with valid paths
+    let metadata = JobMetadata {
+        common: CommonMetadata { process_attempt_no: 0, ..CommonMetadata::default() },
+        specific: JobSpecificMetadata::StateUpdate(StateUpdateMetadata {
+            blocks_to_settle: vec![6, 7, 8], // Gap between 4 and 6
+            snos_output_paths: vec![
+                format!("{}/{}", 6, SNOS_OUTPUT_FILE_NAME),
```

Comment:
This pattern (`"{}/{}"`) is repeated a lot here, it could be really helpful for readability/documentation/DRY to create a util fn for it